okhttp/src/test/java/okhttp3/internal/UtilTest.kt

```diff
@@ -49,33 +52,37 @@ class UtilTest {
         assertThat(checkDuration("timeout", 0, TimeUnit.MILLISECONDS)).isEqualTo(0)
         assertThat(checkDuration("timeout", 1, TimeUnit.MILLISECONDS)).isEqualTo(1)
 
-        assertThatIllegalStateException().isThrownBy { checkDuration("timeout", -1, TimeUnit.MILLISECONDS) }
-            .withMessage("timeout < 0")
-        assertThatIllegalArgumentException().isThrownBy { checkDuration("timeout", 1, TimeUnit.NANOSECONDS) }
-            .withMessage("timeout too small")
-        assertThatIllegalArgumentException().isThrownBy {
-            checkDuration(
-                "timeout",
-                1L + Int.MAX_VALUE.toLong(),
-                TimeUnit.MILLISECONDS
-            )
-        }.withMessage("timeout too large")
+        assertThat(assertThrows<IllegalStateException> {
```

Comment:
This isn't a simple migration.

Reply:
See my other comment. There's an `assertFailure` we can use. Hopefully with structural search/replace.